pkg/southbound/synchronizer/synchronizer.go

```diff
@@ -44,6 +46,9 @@ var log = logging.GetLogger("southbound", "synchronizer")
 
 const matchOnIndex = `(\=.*?]).*?`
 
+type SetNetworkConfig func(targetUpdates map[string]devicechange.TypedValueMap,
```

Comment:
might need to declare this in some new low level package in `utils`, to make sure it has no dependencies other than `api`